guides/cea-and-reimburse.md

```diff
@@ -0,0 +1,9 @@
+# Continuing Education Allowance
```

Comment:
should rename this file to guides/continuing-education-allowance

create another file for guides/expense-reimbursement